adblock_dnsdist/entrypoint.sh

```diff
@@ -9,6 +9,8 @@ run_certbot() {
         --non-interactive --agree-tos \
         --preferred-chain="ISRG Root X1" \
         -d ${TLS_DOMAIN} -m ${TLS_EMAIL};
+
+    chmod -R a+r /etc/letsencrypt/archive
```

Comment:
This is to fix https://github.com/ragibkl/adblock-dns-server/issues/148. Need to wait 2 months to see if it works.